src/Rfc6455/Frame.php

```diff
@@ -433,4 +452,20 @@ public function isValid() : bool
     {
         return !empty($this->opcode);
     }
+
+    /**
+     * The Control Frame is a pong, ping, close frame or a reserved frame between 0xB-0xF.
+     * @see https://tools.ietf.org/html/rfc6455#section-5.5
+     *
+     * @return bool
+     */
+    public function isControlFrame()
+    {
+        return (in_array($this->getOpcode(), [
+            self::OP_PONG,
+            self::OP_CLOSE,
+            self::OP_PING,
+        ])
+        || $this->getOpcode() >= 11 && $this->getOpcode() <= 15);
```

Comment:
This condition is wrong. The Frame is a control frame when the opcode is `>= 8` (which is OP_CLOSE). (and it cannot be more than 15 by nature)

*Also careful with indentation, if this code is wrong, the indentation is also not good and parenthesis are useless.*